branchout-project

```diff
@@ -40,7 +40,7 @@ function projectUpdateError() {
     if test "${projectBranch}" = "HEAD"; then
       echo -e "\\033[0m${2}${projectGroup}/${1}\\033[70D\\033[70C\\033[38;5;202m<rebase>\\033[0m"
     else
-      echo -e "\\033[0m${2}\\033[31m${projectGroup}/${1}\\033[0m \\033[70D\\033[70C\\033[1m $( git pull --rebase origin 2>&1| head -n1)\\033[0m"
+      echo -e "\\033[0m${2}\\033[31m${projectGroup}/${1}\\033[0m \\033[70D\\033[70C\\033[1m $( git pull --ff-only origin 2>&1| head -n1)\\033[0m"
```

Comment:
i like it much better

Reply:
which do you like much better, status quo or the replacement?

Reply:
replacement

Reply:
however we need to have a fetch command and then we need something that shows us pending changes...

Reply:
so status should probably show x commits behind/ahead

Reply:
Agreed on both counts, lots of improvements to come :-)